 iri_wam_controller:
    type: iri_wam_controller/IriWamController
    joint_name: j2_joint 
    pid_parameters:
      p: 100.0
      d: 0.0
      i: 0.0
      i_clamp: 0.0 
 
 iri_wam_cart_controller:
    type: iri_wam_controller/IriWamCartController
    root_name: j1_joint
    tip_name: j7_joint
 
 iri_wam_pr2_controller:
  type: "robot_mechanism_controllers/JointTrajectoryActionController" 
  joints:
    - j1_joint
    - j2_joint
    - j3_joint 
    - j4_joint
    - j5_joint
    - j6_joint
    - j7_joint
  gains:
    j1_joint: {p: 100.0, d: 0.0, i: 0.0, i_clamp: 0.0}
    j2_joint: {p: 100.0, d: 0.0, i: 0.0, i_clamp: 0.0}
    j3_joint: {p: 100.0, d: 0.0, i: 0.0, i_clamp: 0.0}
    j4_joint: {p: 100.0, d: 0.0, i: 0.0, i_clamp: 0.0}
    j5_joint: {p: 100.0, d: 0.0, i: 0.0, i_clamp: 0.0}
    j6_joint: {p: 100.0, d: 0.0, i: 0.0, i_clamp: 0.0}
    j7_joint: {p: 100.0, d: 0.0, i: 0.0, i_clamp: 0.0}
  joint_trajectory_action:
    joints:
      - j1_joint 
      - j2_joint
      - j3_joint
      - j4_joint
      - j5_joint
      - j6_joint
      - j7_joint
    constraints:
      goal_time: 0.6
      j1_joint:
        goal: 0.02
      j2_joint:
        goal: 0.02
      j3_joint:
        goal: 0.02
      j4_joint:
        goal: 0.02
      j5_joint:
        goal: 0.02
      j6_joint:
        goal: 0.02
      j7_joint:
        goal: 0.02  
